app/backend/approaches/chatreadretrieveread.py

```diff
@@ -242,3 +264,17 @@ def get_messages_from_history(
 
         messages = message_builder.messages
         return messages
+    
+    def get_search_query(self, chat_completion: any, user_query: str):
```

Comment:
I think type for chat_completion is a dict, right?

Reply:
Ack. updated.